internal/database/mariadb/init/schema.sql

```diff
@@ -450,3 +450,25 @@ create table if not exists IssueRepositoryService
         foreign key (issuerepositoryservice_service_id) references Service (service_id)
             on update cascade
 );
+
+
+
+create table if not exists ScannerRun
+(
+    scannerrun_run_id          int unsigned primary key auto_increment,
+    scannerrun_uuid            UUID not null unique,
+    scannerrun_tag             varchar(255) not null,
+    scannerrun_start_run       timestamp default current_timestamp() not null,
+    
+    scannerrun_end_run         timestamp default current_timestamp() not null,
+
+    scannerrun_is_completed    boolean not null default false
+);
+
+create table if not exists ScannerRunIssueTracker
```

Comment:
Again an "aggregation" of multiple types. As I wasn't able to understand what this table is supposed to track from the 1st reading, I suggest we also add some comments in the schema file.

Reply:
Again, you are correct, this is part of the 'ScannerRun' protocol documentation, which is out of scope for this partial PR. Once the whole protocol is implemented (and tested), there will be documentation!